src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+        String fullName = lastName + seperator + firstName;
+        if (StringUtils.isNotBlank(middleInitial)) {
+            fullName = fullName + seperator + middleInitial + KFSConstants.DELIMITER;
+        }
+        if(fullName.length() > getPayeeNameFieldSize()) {
+            fullName = StringUtils.substring(fullName, 0, getPayeeNameFieldSize());
+            fullName = removeLastCharacterWhenComma(fullName);
+        }
+        return fullName;
+    }
+    
+    private String removeLastCharacterWhenComma(String fullName) {
+        if (fullName.lastIndexOf(KFSConstants.COMMA) >= getPayeeNameFieldSize()-2) {
+            fullName = fullName.substring(0, fullName.lastIndexOf(KFSConstants.COMMA));
+        }
+        return fullName;
+    }
+
+    @Override
+    public PdpFeedPayeeIdEntry buildPayeeIdEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedPayeeIdEntry payeeIdEntry = new PdpFeedPayeeIdEntry();
+        payeeIdEntry.setContent(line.getEmployeeId());
+        if (StringUtils.equalsIgnoreCase(line.getEmployeeStatus(), ConcurConstants.StandardAccountingExtractPdpConstants.EMPLOYEE_STATUS_CODE)) {
+            payeeIdEntry.setIdType(ConcurConstants.StandardAccountingExtractPdpConstants.EMPLOYEE_PAYEE_STATUS_TYPE_CODE);
+        } else {
+            payeeIdEntry.setIdType(ConcurConstants.StandardAccountingExtractPdpConstants.NON_EMPLOYEE_PAYEE_STATUS_TYPE_CODE);
+        }
+        return payeeIdEntry;
+    }
+
+    @Override
+    public PdpFeedDetailEntry buildPdpFeedDetailEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedDetailEntry currentDetailEntry  = new PdpFeedDetailEntry();
+        currentDetailEntry.setSourceDocNbr(buildSourceDocumentNumber(line.getReportId()));
+        currentDetailEntry.setFsOriginCd(ConcurConstants.StandardAccountingExtractPdpConstants.FS_ORIGIN_CODE);
+        currentDetailEntry.setFdocTypCd(ConcurConstants.StandardAccountingExtractPdpConstants.DOCUMENT_TYPE);
+        currentDetailEntry.setInvoiceNbr("");
+        currentDetailEntry.setPoNbr("");
+        currentDetailEntry.setInvoiceDate(formatDate(line.getBatchDate()));
+        return currentDetailEntry;
+    }
+    
+    @Override
+    public String buildSourceDocumentNumber(String reportId) {
+        String sourceDocNumber = ConcurConstants.StandardAccountingExtractPdpConstants.DOCUMENT_TYPE + reportId;
+        return StringUtils.substring(sourceDocNumber, 0, ConcurConstants.SOURCE_DOCUMENT_NUMBER_FIELD_SIZE);
+    }
+
+    @Override
+    public PdpFeedAccountingEntry buildPdpFeedAccountingEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedAccountingEntry currentAccountingEntry =  new PdpFeedAccountingEntry();
+        currentAccountingEntry.setCoaCd(line.getChartOfAccountsCode());
+        currentAccountingEntry.setAccountNbr(line.getAccountNumber());
+        currentAccountingEntry.setObjectCd(line.getJournalAccountCode());
```

Comment:
I will ping Sherry about that.  I hadn't seen anything about special object handling.